packages/react-components/src/styles/index.ts

```diff
@@ -215,7 +215,7 @@ export default createGlobalStyle<Props & ThemeProps>`
   #root {
     background: ${({ theme }) => theme.bgPage};
     color: ${({ theme }) => theme.color};
-    font-family: sans-serif;
+    font-family: 'Nunito Sans',sans-serif;
```

Comment:
Just wondering out loud - where we have the theme used anyway, maybe it would be a good idea to use it from the theme. 

We obviously have way too many places where we have used sans-serif, so that may cut it down somewhat. (Even if by a handful, still a win)

Reply:
It actually seems to be the bulk - so Just future-proofs nicely.